src/wwwroot/api/scripts/GetExperimentsOfPhysics.js

```diff
@@ -37,74 +82,104 @@ function setHeaders(list, expTable) {
     return columns;
 }	 
 
-
 /* Set All Body Info */
-function setBody(columns, expTable, exp) { 
+function setBody(columns, expTable, exp) {
+    
+    let maxRowSpan = 0;
+
+    //Find max rowSpan
+    
+    for(let poscol = 0; poscol < columns.length; poscol++){
+        if(exp[0][columns[poscol]].length > 0){
+            maxRowSpan = Math.max(exp[0][columns[poscol]].length, maxRowSpan);
+        }
+    }
 
+    //console.log("max row " + maxRowSpan);
+    
     // Traversing the JSON data 
     for (let i = 0; i < exp.length; i++) { 
-        let row = $('<tr/>'); 
-        for (let colIndex = 0; colIndex < columns.length; colIndex++) 
-        { 
-            let val = exp[i][columns[colIndex]]; 
-            
-            // console.log("val : " + val + " size : " + val.length);
-
-            if(columns[0] === "চাপ" && colIndex === 0) {
-                let radioString = ' ';
+        //let row = $('<tr/>'); 
+        let rowStr = "<tr>";
 
-                console.log("Nth radio checked : " + val);
+        if(maxRowSpan === 0){
+            for (let colIndex = 0; colIndex < columns.length; colIndex++){
+                rowStr += "<td>"+exp[i][columns[colIndex]]+"</td>";
+            }
+            rowStr += "</tr>";
+        }
+        else {
 
-                if(val === 0){
-                    radioString += '<label><input type="radio" name="group'+i+'" checked/> <span>বায়ু মন্ডলের চাপ</span> </label>' + '<br>';
-                    radioString += '<label><input type="radio" name="group'+i+'"/> <span>বায়ু মন্ডলের চাপের বেশী</span> </label>' + '<br>';
+            let value = exp[i][columns[0]];
+            let radioString = "";
+            //চাপ
+            if(columns[0] === "চাপ") {
```

Comment:
Needs to make it dynamic. **Putting it into future factory.**

Reply:
I need to some time's then I hope, I can handle this dynamically